client/app/scripts/charts/edge-container.js

```diff
@@ -50,14 +56,9 @@ class EdgeContainer extends React.Component {
   }
 
   render() {
-    const { layoutPrecision, points } = this.props;
+    const { layoutPrecision } = this.props;
     const other = _.omit(this.props, 'points');
 
-    if (layoutPrecision === 0) {
```

Comment:
This is needed for really busy topologies. It returns a non-animated edge, which otherwise would look yanky.